src/domain/course/CoursePageContainer.tsx

```diff
@@ -55,7 +81,11 @@ const CoursePageContainer: React.FC = () => {
                 <EventClosedHero />
               ) : (
                 <>
-                  <EventHero event={course} eventType="course" />
+                  <EventHero
+                    event={course}
+                    superEvent={superEvent?.courseDetails}
```

Comment:
I was thinking that what if we would pass superEventId to the `EventHero` and doing query inside the EventHero? So we would avoid duplicate code in `CoursePageContainer` and `EventPageContainer`. Another option to avoid duplicate code is to make custom hook for loading superEvent and use it in both files.

Reply:
well, its extra query, but we might need the superevent data in other components. @Sartonon, what do you think?

Reply:
we have the same duplicate query of the event itself basically